trulens/nn/models/pytorch.py

```diff
@@ -269,7 +269,9 @@ def fprop(
         else:
             doi_repeated_batch_size = intervention[0].shape[0]
             batched_model_args = []
+            batched_model_kwargs = {}
 
+            # tile batches for interpolation
             for val in model_args:
                 doi_resolution = int(doi_repeated_batch_size / val.shape[0])
                 tile_shape = [1 for _ in range(len(val.shape))]
```

Comment:
Just a note, depending on how the memory task goes, we may want to do smarter checking and not tile everthing. We could check if doi_resolution is the expected length.
captured here: https://dev.azure.com/ailens-io/truera/_backlogs/backlog/truera%20Team/Epics/?workitem=5907

Reply:
Did this, the first arg is expected to be batchable, all subsequent ones are batched only if they match the first's first dim.